lib/cli.ex

```diff
@@ -22,24 +22,26 @@ defmodule Onigumo.CLI do
 
             case switches do
               [] -> module.main(working_dir)
-              _ -> usage_message()
+              _ -> usage_message("invalid OPTIONS #{inspect(OptionParser.to_argv(switches))}")
             end
 
           :error ->
-            usage_message()
+            usage_message("invalid COMPONENT #{inspect(component)}")
         end
 
-      {_, _, [_ | _]} ->
-        usage_message()
+      {_, _, invalid} when invalid != [] ->
```

Comment:
I found out that the guard can be omitted.

```suggestion
      {_, _, invalid = [_ | _]} ->
```

keeps the original pattern, only attaching a name to it.